test/multest.jl

```diff
@@ -20,4 +20,10 @@ qv = QuArray(v)
 @assert qv'*qv == dot(v,v)
 @assert rawcoeffs(qv*qv') == v*v'
 
-@assert qv'*qm*qv == first(v'*m*v)
\ No newline at end of file
+@assert qv'*qm*qv == first(v'*m*v)
+
+@assert rawcoeffs(2*qv) == 2*v
+@assert rawcoeffs(qv/2) == v/2
+@assert rawcoeffs(scale(2,qv)) == 2*v
+@assert norm(qv) == norm(v)
+#@assert rawcoeffs(normalize(qv)) == rawcoeffs(qv)/norm(v)
```

Comment:
normalize(qv).coeffs = [0.2672612419124244,0.5345224838248488,0.8017837257372732]
qv.coeffs/norm(v) = [0.2672612419124244,0.5345224838248488,0.8017837257372733]
Hence the test fails.